src/arch/Sound/RageSoundDriver_WaveOut.cpp

```diff
@@ -20,13 +20,13 @@
 REGISTER_SOUND_DRIVER_CLASS( WaveOut );
 
 const int channels = 2;

```

Comment:
Let's pull all these constants in an anonymous namespace and let's use MACRO_NAMING for the variable names.